apps/api/src/app/events/services/mail-service/handlers/emailjs.handler.ts

```diff
@@ -0,0 +1,23 @@
+import { EmailJsConfig } from '@notifire/emailjs/build/main/lib/emailjs.config';
+import { EmailJsProvider } from '@notifire/emailjs';
+import { ChannelTypeEnum } from '@notifire/shared';
+import { ICredentials } from '@notifire/dal';
+import { BaseHandler } from './base.handler';
+
+export class EmailJsHandler extends BaseHandler {
+  constructor() {
+    super('emailjs', ChannelTypeEnum.EMAIL);
+  }
+  buildProvider(credentials: ICredentials, from: string) {
+    const config: EmailJsConfig = {
+      from,
+      host: credentials.host,
+      port: Number(credentials.port),
+      secure: credentials.secure,
+      user: null,
```

Comment:
Any specific reason the user is null here? It's a different concept from the $user_id we use internally, here a user is the email box username as in user@host.com

Reply:
No specific reason it's my typo, it should be added as credentials because this is the user name the user used on this provider.